app/views/docs/production.phtml

```diff
@@ -13,38 +13,48 @@
 
 <h2><a href="/docs/production#emails" id="emails">Emails</a></h2>
 
-<p>Sending emails is hard. There are a lot of SPAM rules and configurations to master in order to set a functional SMTP server. The SMTP server that comes packaged with Appwrite is great for development but needs some work done to function well against SPAM filters.</p>
+<p>Sending emails is hard. There are a lot of spam rules and configurations to master to set up a functional SMTP server. The SMTP server that comes pre-packaged with Appwrite is satisfactory for testing, but you'll need a third-party solution to deliver emails that do not get labeled as spam.</p>
 
-<p>Another easier option is to use an ‘SMTP as a service’ product like Sendgrid or Mailgun. You can <a href="/docs/email-delivery">change Appwrite SMTP settings</a> and credentials to any 3rd party provider you like who support SMTP integration using our Docker environment variables. Most SMTP providers offer a decent free tier to get started with.</p>
+<p>You can self-host an SMTP service or use one of the "SMTP as a service" products like Sendgrid or Mailgun. You can <a href="/docs/email-delivery">change Appwrite SMTP settings</a> and credentials to any 3rd party provider you like who support SMTP integration using our Docker environment variables. Most SMTP providers offer a decent free tier to get started with.</p>
 
 <h2><a href="/docs/production#backups" id="backups">Backups</a></h2>
 
-<p>Backups are highly recommended for any production environment. Currently, there is no built-in script we provide to do this automatically. To be able to backup your Appwrite server data, stats, and files, you will need to do the following.</p>
+<p>Backups are highly recommended for any production environment. Currently, there is no built-in script we provide to do this automatically. You must do the following to back up your Appwrite server data, stats, and files.</p>
 
-<ol>
+<ol class="margin-top margin-bottom-large text-size-normal">
     <li>Create a script to backups and restore your MariaDB Appwrite schema. Note that trying to backup MariaDB using a docker volume backup can result in a corrupted copy of your data. It is recommended to use MariaDB or MySQL built-in tools for this.</li>
     <li>Create a script to backups and restore your InfluxDB stats. If you don’t care much about your server stats, you can skip this.</li>
     <li>Create a script to backup Appwrite storage volume. There are many online resources explaining different ways to backup a docker volume. When running on multiple servers, it is very recommended to use an attachable storage point. Some cloud providers offer integrated backups to such attachable mount like GCP, AWS, DigitalOcean, and the list continues.</li>
```

Comment:
It would be worth mentioning to also backup `.env`and `docker-compose.yml`. ENV holds secrets and server configuration. docker-compose holds container configuration and Appwrite versions.

Reply:
+1

Reply:
Done